setuptools/config.py

```diff
@@ -399,6 +401,20 @@ def parse(self):
 
             section_parser_method(section_options)
 
+    def deprecated_config_handler(self, msg, warning_class, func):
```

Comment:
This is a bit of a nitpick, but I'd prefer if `func` came first in this case.

Reply:
Also can you name this `_deprecated_config_handler`? We do not want this to be considered part of the public interface.

Reply:
Sure, let me make that change.